.JuliaFormatter.toml

```diff
@@ -0,0 +1,10 @@
+style = "sciml"
```

Comment:
should we rename this to meshes?

Reply:
@kylebeggs are these styles fixed in the template or the name can vary freely?

Reply:
This grabs the pre-built style named "sciml". You can create your own style with your own name, but I am not familiar with how to do it. I just read in the docs that it is possible.

Reply:
We should try to create our own style before merging then 👍🏽

Reply:
Ok, we should label this to solicit for help because I'm not sure how to do that and I don't have the time right now.

Reply:
There are three available styles in JuliaFormatter.jl: [YAS](https://domluna.github.io/JuliaFormatter.jl/dev/yas_style/), [Blue](https://domluna.github.io/JuliaFormatter.jl/dev/blue_style/) and [SciML](https://domluna.github.io/JuliaFormatter.jl/dev/sciml_style/).

If we want our own named style, we need to create a repository that describes our style (e.g. [Blue style](https://github.com/invenia/BlueStyle)) and a PR to JuliaFormatter.jl to support our style. I think this is too hard and not what we would like to have in this PR.

We can just use the default or some style, and modify it by adding configs like `indent = 2`.

Reply:
Thank you @hyrodium, we will take a look at the existing styles and see if we can simply adopt one and adjust a few options.